components/automate-ui/src/app/modules/infra-proxy/infra-search-bar/infra-search-bar.component.html

```diff
@@ -1,17 +1,20 @@
-<div class="search-wrapper" tabindex="0">
-  <div class="search-input-wrapper">      
-    <input #search_box chefInput
-      id="search_box"
-      aria-label="Search"
-      (keyup)="handleInput($event.key, search_box.value)"
-      [value]="inputText"
-      type="text"
-      class="search-input"
-      placeholder="Search {{ placeHolder }}"
-      autocomplete="off">
-    <button chef-button class="search-button primary" primary (click)="handleFiltersClick(search_box.value)">
-      <chef-icon>search</chef-icon>
-    </button>
-  </div>
-  <ng-content></ng-content>
-</div>
+<form role='search'
```

Comment:
Author note: Accessibility best practices tell us that search bars should be in the form tag.

Source: http://web-accessibility.carnegiemuseums.org/code/search/#:~:text=Best%20Practices&text=A%20search%20bar%20should%20be,necessary%20to%20use%20the%20form.&text=Also%2C%20always%20include%20the%20word,and%20in%20the%20submit%20button.